recipes/bkcharts/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "bkcharts" %}
+{% set version = "0.2" %}
+{% set sha256 = "a5eaa8e78853dcecaa46345812e4fabe9cd3b96330ebf0809f640a4a0556d72e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - six >=1.5.2
+    - numpy >=1.7.1
+    # xref: https://github.com/bokeh/bkcharts/issues/72
+    - pandas
+
+test:
+  requires:
+    # Seems the requires bokeh to function. However bokeh
+    # requires this package too. Not expecting this to be
+    # a long term thing, but it is an issue we have to
+    # workaround for testing somehow.
+    #
+    # xref: https://github.com/bokeh/bkcharts/issues/73
+    - bokeh ==0.12.6
```

Comment:
So this is a can of worms. `bokeh` requires `bkcharts`, but `bkcharts` uses `bokeh` throughout. Thus import tests fail without `bokeh`. It seems like `bokeh` will drop `bkcharts` in the long term. Would expect `bkcharts` also starts depending on `bokeh` then. In the interim, this helps us avoid a painful cyclic dependency.

Also it seems this won't work with the current `bokeh` in `conda-forge` (0.12.5). So am pinning `bokeh` to 0.12.6.

xref: https://github.com/bokeh/bkcharts/issues/73